presto-tests/src/test/java/io/prestosql/execution/TestCoordinatorDynamicFiltering.java

```diff
@@ -93,11 +95,14 @@ public void setup()
         // create lineitem table in test connector
         getQueryRunner().installPlugin(new TestPlugin());
         getQueryRunner().installPlugin(new TpchPlugin());
+        getQueryRunner().installPlugin(new TpcdsPlugin());
         getQueryRunner().installPlugin(new MemoryPlugin());
         getQueryRunner().createCatalog("test", "test", ImmutableMap.of());
         getQueryRunner().createCatalog("tpch", "tpch", ImmutableMap.of());
+        getQueryRunner().createCatalog("tpcds", "tpcds", ImmutableMap.of());
         getQueryRunner().createCatalog("memory", "memory", ImmutableMap.of());
         computeActual("CREATE TABLE lineitem AS SELECT * FROM tpch.tiny.lineitem");
+        computeActual("CREATE TABLE store_sales AS SELECT * FROM tpcds.tiny.store_sales");
```

Comment:
why not existing table from tpch?

Reply:
I didn't see NULLs in tpch tables, i'm ok to use a table from there if there are NULLs in some table in it.

Reply:
IIRC, TPC-H tables do not contain NULL values...

Reply:
Does it increase test runtime?

Reply:
It adds a few seconds, but not too much.